TableGenerator/Reference.fs

```diff
@@ -3,111 +3,187 @@
 open System
 open TableGenerator.Shared
 
-let referenceTemplate: ReferenceTemplate =
-    """[{0}-badge-{1}]: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/{2}/{3}_Release_version_badge.svg
+// https://aka.ms/dotnet/net5/dev/Sdk/dotnet-sdk-win-x64.exe
+
+// https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/dotnet-sdk-latest-win-x64.exe
+// https://aka.ms/                         dotnet/net5/preview2/Sdk/dotnet-sdk-win-x64.exe
+
+// https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/win_x64_Release_version_badge.svg
+// https://aka.ms/                         dotnet/net5/preview2/Sdk/win_x64_Release_version_badge.svg
+
+//  https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/latest.version
+//  https://aka.ms/dotnet/net5/preview2/Sdk/latest.version
```

Comment:
nit: you can replace these comments with unit tests. And unit tests won't out of date